class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
I wasn't sure if the global variable `$blog_id` is set properly by the runner and set it just in case. If it is, then I'll remove the site id. Do you know that for sure?